### PR TITLE
Restore owners for ^features

### PR DESCRIPTION
**Pull request by iranzo**
# Description

Read owners for ^/features in this repo

Fixes # (issue)

## Type of change

Please select the appropiate options:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Testing

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:

- Versions:
- Hardware:

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] If a change is adding a feature, it should require a change to the README.md and the review should catch this.
- [ ] If the change is a fix, it should have an issue. The review should make sure the comments state the issue (not just the number) and it should use the keywords that will close the issue on merge.
- [ ] A change should not being merged unless it passes CI or there is a comment/update saying what testing was passed.
- [ ] PRs should not be merged unless positively reviewed.

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -22,3 +22,10 @@ filters:
   "^ansible-ipi-install/.*":
     labels:
       - kind/installer
+
+  "^features":
+    reviewers:
+      - code-reviewers
+      - cnf-reviewers
+    approvers:
```
Comment (slintes) [inline OWNERS:30]: this won't work, that's why we have https://github.com/openshift-kni/baremetal-deploy/blob/master/features/OWNERS, see https://github.com/openshift-kni/baremetal-deploy/pull/64

Comment (iranzo) [reply OWNERS:30]: Hi Mark, should we add them for the global filter then?

We want to limit bugging others when putting changes on docs and the ansible script folders as almost 99% of the time, prow assigned 'others' which are not relevant to the repo.

Any advice here?

Comment (slintes) [reply OWNERS:30]: keep the aliases file in the root directory, but put reviewers and approvers into  /features/OWNERS... that *should* work

Comment (slintes) [reply OWNERS:30]: or reviewers with filter and approvers in its own file, as you did in the new PR, should also work

**Pull request merged**